src/main/java/com/ericsson/eiffel/remrem/semantics/SemanticsService.java

```diff
@@ -261,4 +278,37 @@ public ValidationResult validateMsg(String msgType, JsonObject jsonvalidateMessa
         }
         return validationResult;
     }
+    
```

Comment:
is this correct?

Reply:
It's a legacy code. Not a part of the story.

Reply:
It's a legacy code. Not a part of the story